internal/authenticator/authenticator.go

```
@@ -224,31 +238,32 @@ func (a *Authenticator) AuthenticateHttpRequest(req *http.Request, xCustomProvid
 		return nil, nil, err
 	}
 
-	hash := hasher.Hash(raw)
+	hash, err := a.getHashViaSecondary(raw)
```

golangci-lint (2.12.2)

[error] ineffectual assignment to err

(ineffassign)